mindjunkies/courses/models.py

```
@@ -50,6 +50,7 @@ class Course(BaseModel):
     total_rating = models.DecimalField(max_digits=5, decimal_places=2, default=0.0)
     number_of_ratings = models.PositiveIntegerField(default=0)
     number_of_enrollments = models.PositiveIntegerField(default=0)
+    
```

Ruff (0.8.2)

Blank line contains whitespace

Remove whitespace from blank line

(W293)